src/views/settings/Msg/index.vue

```diff
@@ -28,7 +28,7 @@ export default {
         {
           title: this.$t('Msg Template'),
           name: 'MsgTemplate',
-          hidden: !this.$hasPerm('settings.change_email')
+          hidden: !this.$hasPerm('settings.change_email') || !this.$store.getters.hasValidLicense
         },
         {
           title: this.$t('SMS'),
```

Comment:
The code does not show any significant differences between version 2021-09 and current date 2025-09. It seems to be an initial setup without actual functional changes or optimizations. Since there is no indication of errors or specific areas needing attention, I cannot recommend any modifications here. If you're looking for help with specific functionalities or improvements within that section, please let me know!

Would you like further assistance related to your original inquiry?